ngx_http_sxg_filter_module.c

```diff
@@ -540,6 +574,69 @@ bool is_valid_config(ngx_conf_t* nc, const ngx_http_sxg_srv_conf_t* sc) {
   return valid;
 }
 
+static
+bool append_header(ngx_list_t* headers, const char* key, const char* value) {
+  ngx_table_elt_t* h = ngx_list_push(headers);
+  if (h == NULL) {
+    return false;
+  }
+
+  h->hash = 1;
+  h->key.len = strlen(key);
+  h->key.data = (u_char *) key;
+  h->value.len = strlen(value);
+  h->value.data = (u_char *) value;
+  return true;
+}
+
+static ngx_int_t ngx_http_cert_chain_handler(ngx_http_request_t* req) {
+  // Check the URL is certificate request.
+  ngx_http_sxg_srv_conf_t* ssc =
+      ngx_http_get_module_srv_conf(req, ngx_http_sxg_filter_module);
+  if (ssc->cert_path.len > 0 && req->uri.len == ssc->cert_path.len &&
+      ngx_memcmp(req->uri.data, ssc->cert_path.data, req->uri.len) == 0) {
+    bool refreshed = refresh_if_needed(&ssc->cert_chain);
+    if (refreshed) {
+      ngx_log_error(NGX_LOG_ERR, req->connection->log, 0,
```

Comment:
Should the logging constant be `NGX_LOG_INFO` or `NGX_LOG_DEBUG`?

Reply:
OK, `NGX_LOG_INFO` would be suitable for this purpose.